browser-test/test.endtoend.ts

```diff
@@ -53,34 +53,29 @@ describe('Run tests against gRPC server', () => {
   const client = new EchoClient(opts);
 
   before(async function() {
-    this.timeout(30000);
+    this.timeout(40000);
 
-    //TODO: Make sure server is up and running before starting tests, but
-    //      without using sleep for some arbitrary amount of time
-    await sleep(10000);
```

Comment:
Can you please remove the `sleep` function from above? We don't need it anymore.

Reply:
I think it's used in the before() block; if the server isn't up yet, we wait for one second before trying again